.github/workflows/deploy_devnet.yml

```diff
@@ -0,0 +1,97 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: LGPL-3.0-only
+
+name: spectre_devnet
```

Comment:
We are going to have just testnet for now.

Reply:
I think it is best if it deploys on every commit for now (same as devnet).
Sorry if I didn't specify that earlier.

Reply:
resolved!